public/plugin.tsx

```diff
@@ -160,8 +170,12 @@ export class AssistantPlugin
     setChrome(core.chrome);
     setNotifications(core.notifications);
 
-    return {};
+    return {
+      ...this.dataSourceService.start(),
```

Comment:
```suggestion
      dataSourceServiceStart: this.dataSourceService.start(),
```
Same here, I'd vote for a single clear entry.